### PR TITLE
feat: publish @geprog/vite-plugin-env-config

### PR DESCRIPTION
**Pull request by lukashass**

### CODE REVIEW

**Review by anbraten — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const backendURL = getEnvConfig('BACKEND_URL');
+```
+
+For production use `envsubst` as outlined in the [next section](#motivation).
+
+## Motivation
+
+To adhere to the principles of the [twelve-factor app](https://12factor.net/config)
+you might want use a config file included in your `index.html` which is generated by `envsubst`
+when your frontend server starts:
```
[the remark this answers is not in the document]
Comment (lukashass) [reply README.md]: changed in c162dfb292ef1763545cbd055d1ae1d0e88e2f80

File: src/getEnvConfig.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export interface EnvConfig {
+  [key: string]: string;
+}
+
+export function getEnvConfig<T extends keyof EnvConfig>(key: keyof EnvConfig): EnvConfig[T] | undefined {
+  const { env } = window as unknown as { env: EnvConfig };
```
[the remark this answers is not in the document]
Comment (lukashass) [reply src/getEnvConfig.ts:6]: I'll do this in #5

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (anbraten) [reply README.md]: nice thx.

**Review by anbraten — APPROVED** [review]
lgtm

**Pull request merged**